usrsctplib/netinet/sctp_callout.h

```diff
@@ -99,20 +87,26 @@ typedef struct sctp_callout sctp_os_timer_t;
 
 #define	SCTP_CALLOUT_ACTIVE	0x0002	/* callout is currently active */
 #define	SCTP_CALLOUT_PENDING	0x0004	/* callout is waiting for timeout */
-
-void sctp_os_timer_init(sctp_os_timer_t *tmr);
+#define	SCTP_CALLOUT_CANCELLED	0x0008	/* callout is cancelled */
```

Comment:
Note for reviewers:
I've decided to replace cancellation flag of current timer implemented as global variable [`sctp_os_timer_waiting`](https://github.com/sctplab/usrsctp/blob/913de3599feded8322882bdae69f346da5a258fc/usrsctplib/netinet/sctp_callout.c#L88) with a state stored in timer itself.